### DISCUSSION TITLE
Sending a voice message during /start

### DISCUSSION THREAD

**Discussion by danielatlarge** [python-telegram-bot/python-telegram-bot#3638] [Q&A]
I would like to send an AI crafted voice message at the beginning of /start
The untouched code looks like this:

```python
async def start_handle(update: Update, context: CallbackContext):
    await register_user_if_not_exists(update, context, update.message.from_user)
    user_id = update.message.from_user.id
    
    db.set_user_attribute(user_id, "last_interaction", datetime.now())
    db.start_new_dialog(user_id)
    
    reply_text = """Hello, and welcome to my bot! \n\n"""
    reply_text += HELP_MESSAGE

    await update.message.reply_text(reply_text, parse_mode=ParseMode.HTML)
```

Wanted to integrate something like this: `await send_voice(chat_id=chat_id, voice="https://example.com/hello.mp3")`
Can anyone steer me in the right direction?

**Comment by Bibo-Joshi** [comment]
Hi. I don't understand the question, i.e. what exactly you're not getting to work. Please have a look at [this wiki page](../wiki/Ask-Right).
